modules/developers-guide/pages/tutorials/phonebook.adoc

```diff
@@ -89,48 +85,48 @@ To start the network locally:
 * You should now have **two terminals** open.
 * You should have the **project directory** as your **current working directory**.
 --
-. Start the {IC} network on your local computer by running the following command:
+. Start the local canister execution environment on your local computer by running the following command:
 +
 [source,bash]
 ----
 dfx start --clean
 ----
 +
-For this tutorial, we're using the `+--clean+` option to start the {IC} network in a clean state.
+For this tutorial, we're using the `+--clean+` option to start the local canister execution environment in a clean state.
 +
-This option removes any orphan background processes or canister identifiers that might disrupt normal operations.
+This option removes any orphan background processes or canister smart contract identifiers that might disrupt normal operations.
```

Comment:
I think I'd leave "canister identifiers" since it's a fixed term – and using canister is actually fine in a context where it is clear that we refer to a smart contract.

Reply:
Makes sense.
```suggestion
This option removes any orphan background processes or canister identifiers that might disrupt normal operations.
```